crates/valence_stresser/src/args.rs

```diff
@@ -0,0 +1,24 @@
+use clap::{arg, command, Parser};
+
+#[derive(Parser)]
+#[command(author, version, about)]
+pub(crate) struct StresserArgs {
+    /// IPv4/IPv6/DNS address of a server.
+    #[arg(short = 't', long = "target")]
+    pub target_host: String,
+
+    /// Number of connections to the target.
+    #[arg(short = 'c', long = "count")]
+    pub connections_count: usize,
+
+    /// Name prefix of connections.
+    #[arg(default_value = "Stresser")]
+    #[arg(short = 'n', long = "name")]
+    pub name_prefix: String,
+
+    /// Spawn cooldown of connections in milliseconds.
+    /// The lower the value, the more frequently connections are spawned.
+    #[arg(default_value = "100")]
```

Comment:
The default should be lower. Maybe 10?

Reply:
Actually, it depends on how powerful the hardware on which the stresser is run. So there is a need to find out the average value that would fit most hardware. If the value is too low, there is a high potential for timed-out disconnections from a server due to overload.